Tone/instrument/CymbalSynth.js

```diff
@@ -0,0 +1,304 @@
+	 *  @param  {Time} [time=now] the time the cymbal will release
+	 *  @returns {Tone.CymbalSynth} this
+	 *  @example
+	 *  //set the release to a small amount for a realistic choke
+	 *  cymbal.body.envelope.release = 0.2;
+	 *  cymbal.triggerAttack("0:0:0");
+	 *  cymbal.triggerRelease("0:0:0" + 0.4);
+	 */
+	Tone.CymbalSynth.prototype.triggerRelease = function(time){
+		this.strike.envelope.triggerRelease(time);
+		this.body.envelope.triggerRelease(time);
+		return this;
+	};
+
+	/**
+	 * Clean up.
+	 * @return {Tone.CymbalSynth} this
+	 */
+	Tone.CymbalSynth.prototype.dispose = function(){
+		Tone.Instrument.prototype.dispose.call(this);
+		this.harmonicity.dispose();
+		this.harmonicity = null;
+		this.frequency.dispose();
+		this.frequency = null;
+		this.inharmRatios = null;
+		this.harmRatios = null;
+
+		for(var i = 0; i <6; i++){
+			this._oscillators[i].dispose();
+			this._scaledSignals[i].dispose();
+			this._freqMult[i].dispose();
+		}
+		this._oscillators = null;
+		this._scaledSignals = null;
+		this._freqMult = null;
+
+		this.body.dispose();
+		this.strike.dispose();
+		this.strike = null;
+		this.body = null;
+		return this;
+	};
+
+	/**
+	 * Cymbal part helper class
+	 * @private
+	 */
+	var CymbalComponent = function(options){
+		this.output = this.context.createGain();
+		this.volume = new Tone.Signal({
+			"param" : this.output.gain, 
+			"units" : Tone.Type.Decibels,
+			"value" : options.volume
+		});
+		this._readOnly(["volume"]);
+		this._highPass = new Tone.Filter({
+			"type" : "highpass"
+		}).connect(this.output);
+		this.envelope = new Tone.AmplitudeEnvelope({
+			"attack" : options.envelope.attack,
+			"decay" : options.envelope.decay,
+			"sustain" : 0,
+			"release": 0,
+			"attackCurve" : "exponential"
+
+		}).connect(this._highPass);
+		this.input = new Tone.Filter({
+			"type" : "bandpass",
+			"Q" : options.Q
+		}).connect(this.envelope);
+		this.cutoff = new Tone.Signal(options.cutoff).connect(this.input.frequency);
+		this._highPassFrequency = new Tone.Multiply(options.frequencyScalar).connect(this._highPass.frequency);
+		this.cutoff.connect(this._highPassFrequency);
+		this.resonance = new Tone.Signal(options.resonance).connect(this.input.Q);
+		this._highPassResonance = new Tone.Multiply(options.resonanceScalar).connect(this._highPass.Q);
+		this.resonance.connect(this._highPassResonance);
+	};
+
+	CymbalComponent.prototype.dispose = function(){
+		this.output.dispose();
+		this.output = null;
+		this.volume.dispose();
+		this.volume = null;
+		this._highPass.dispose();
+		this._highPass = null;
+		this.envelope.dispose();
+		this.envelope = null;
+		this.input.dispose();
+		this.input = null;
+		this.cutoff.dispose();
+		this.cutoff = null;
+		this._highPassFrequency.dispose();
+		this._highPassFrequency = null;
+		this.resonance.dispose();
+		this.resonance = null;
+		this._highPassResonance.dispose();
+		this._highPassResonance = null;
+		return this;
+	};
+
+	Tone.extend(CymbalComponent);
```

Comment:
This line needs to be above `CymbalComponent.prototype.dispose`, otherwise that method will just be overwritten.